### DISCUSSION TITLE
Adding Media [TV] - Unable to expand first row of seasons

### DISCUSSION THREAD

**Discussion by photo-copy** [chrisbenincasa/tunarr#1017] [General]
When I try to add a TV show to a channel, and that show has 2 rows for display, I am unable to expand the seasons on the first row to select specific episodes. The second row works as expected. If I click on Season 1-8 (or 1-7 with specials), nothing happens. 

I have cleared my cache, and deleted all Tunarr & Jellyfin related cookies. Restarted my browser (Firefox). I tried this in Edge, and was able to recreate it. I also tried on a different PC and was able to replicate it. I have also rebooted my Windows 11 PC.

I have Version: 0.16.11 installed in a Docker container on Unraid.

Let me know if if I should open an issue.

**Comment by chrisbenincasa** [comment]
Hey thanks for the report - just want to clarify that you're talking about the list view and not the grid view, right?

**Comment by photo-copy** [comment]
Sorry for the delay, it is happening in the grid view. The list view works as it should (which is a suitable workaround, as far as I am concerned).

**Comment by chrisbenincasa** [comment]
I suppose that implies the issue is on the front end however they use
almost identical code. Also which version are you running? I can tell from
some of the logs it’s probably a few behind

**Comment by photo-copy** [comment]
At the time, I was running version: 0.16.11 installed in a Docker container on Unraid. I just upgraded to 0.16.13, and the issue is still present.

**Comment by chrisbenincasa** [comment]
Oops sorry. I think I responded to the wrong issue here!
